common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -231,7 +231,7 @@ public static boolean equalIncreasingByteBuffer(int start, int len, ByteBuffer b
       return false;
     }
     buf.rewind();
-    if (buf.remaining() != len) {
+    if (buf.remaining() < len) {
```

Comment:
I made this change so that the behavior of equalIncreasingByteBuffer and equalIncreasingByteArray is consistent. It seems strange that one requires the sequence to match the given length while the other one requires the sequence to be at least of the given length.